homeassistant/components/plugwise/climate.py

```diff
@@ -67,17 +67,17 @@ def __init__(
             self._attr_preset_modes = presets
 
         # Determine hvac modes and current hvac mode
-        self._attr_hvac_modes = [HVACMode.HEAT]
+        self._attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT]
```

Comment:
Plugwise doesn't support "off" natively?

Reply:
For an Adam with and OpenTherm device, Plugwise does. The regulation_mode off could be used for this:
![image](https://github.com/home-assistant/core/assets/11290930/2aff6b71-33a9-49b6-95e2-224b36ac1529)
We could add the `mode = off` option for configurations that provide this option. We would need to investigate a bit more how to implement the setting to off-mode.

An Anna connected to an OpenTherm device does not provide such an option.

For other Anna-configurations, with city-heating via a valve/valves, OnOff-devices, we would need to investigate.

Reply:
It can be done but it will be getting a little complex. A Select has to be changed and later changed back to what it was.

Reply:
Realizing this will be complex, a Select needs to be set to `off` and later back to its previous state.
This previous state has to be saved in HA and survive a HA restart. This previous state cannot be provided by the backend because a restart will also erase the previous state in the backend.

Reply:
As an alternative, the Google Assistant integration could add the HVACMode.OFF when a thermostat does not have this mode. This would correct the operation in Google Home for all thermostats without the HVACMode.OFF.